### PR TITLE
Feature Updates

### PR DESCRIPTION
**Pull request by AzukaChan**
Refresh Torrent Data
----------------
Added a button to manually refresh torrent seed/leech data. There is currently no limit on this so you probably shouldn't go spamming it.

Category Overview
-----------------
Added a 'Category Overview' which can be accessed by clicking a category from the index page. Once a category reaches 10 torrents; previous/next buttons will appear at the bottom of the page allowing users to navigate through pages.

Basic Account/Admin Control Panel
------------
Added a really basic acp menu that doesn't really do much at the moment. To give yourself admin access, simply set your 'uploaderstatus' to 99 in the database and navigate to the preferences page from the top right corner once you login. This will allow you to delete torrents from any user.

If your 'uploaderstatus' is not set to 99 you will receive a regular 'Account Control Panel' page

### CODE REVIEW

File: en/view/acp_delete_file.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+	include_once "../../php/libs/database.php";
+    
+	date_default_timezone_set('Europe/Brussels');
+
+	$db = new Db();
+
+	$db->query("DELETE FROM `torrents` WHERE `id`=".$_GET['delete_id']."");
```
Comment (KevinJDurant) [inline en/view/acp_delete_file.php]: What's the security on this? Can anyone delete any torrent file?

File: en/view/category.php [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+			<?php } ?></ul>
+</nav>
+			<?php } ?>
+        <hr>
+
+        <!-- Footer -->
+        <footer>
+            <div class="row">
+                <div class="col-lg-12">
+                    <p>Happy Downloading~</p>
```
Comment (KevinJDurant) [inline en/view/category.php]: Copyright &copy; Your Website 2019

**Comment by KevinJDurant** [comment]
I see you have solved some branding issues in the next commit. But I believe you've missed some, sorry for being this strict. I can solve it myself but I only have time early next week.

Thanks for the amazing contributions so far!

File: en/view/acp_delete_file.php (hunk 1 above)
Comment (KevinJDurant) [reply en/view/acp_delete_file.php]: I believe I had a simple function 'valid_user' in upload.php to see if the logged in user has a valid session key. It's at least some security, I didn't know much about this when I originally made this project.

Comment (AzukaChan) [reply en/view/acp_delete_file.php]: Only users with 'uploaderstatus'=99 will be able to delete torrents from the acp menu. This is still very basic; I haven't added previous next buttons so the page might be huge depending on how many torrents you have on your site. Will add it on the next commit.

For anyone other than an admin; torrents will not appear on acp as they can delete their own torrents from my-torrents.php

File: en/view/category.php (hunk 2 above)
Comment (AzukaChan) [reply en/view/category.php]: Changed 'Happy Downloading~' to 'Copyright © Your Website 2019'

File: en/view/acp_delete_file.php (hunk 1 above)
Comment (AzukaChan) [reply en/view/acp_delete_file.php]: Ahh yes I see; adding that would make things a little bit more secure.
Would you be able to suggest edit those things for me? I'm learning PHP as I go with this project.

Comment (KevinJDurant) [reply en/view/acp_delete_file.php]: @AzukaChan I'll check it out during the week, busy schedule. I'll see which kind of security we can set on this API. If you want and you use Discord, you can always mail me your tag#numbers to contact@kevindurant.be.

**Comment by AzukaChan** [comment]
http://i.imgur.com/jXDEeEv.png

Users with uploaderstatus 99 won't appear in the list just to avoid accidentally removing your own admin status.

**Pull request merged**